data/gmh/Holz_1897_Laurin/stemma.gv

```diff
@@ -0,0 +1,40 @@
+digraph {
+
+KM [label="", color="grey"]
+x [label="x¹)", color="grey"] # The footnote refers to the use of the symbol in the critical apparatus of the edition and has no meaning for the stemma.
+y [color="grey"]
+w [color="grey"]
+zh [label="z", color="grey"]
+A [color="grey"]
+B [color="grey"]
+C [color="grey"]
+D [color="grey"]
+Hpz [label="", color="grey"]
+
+
+A -> x
+x -> y
+x -> zh
+x -> w
```

Comment:
Weird, this `w` lost node without descent. Any idea what it means ?

Reply:
It was just a mistake, w is not hypothetical.